modules/ROOT/pages/auto-upgrade-runtime-version-used.adoc

```diff
@@ -0,0 +1,12 @@
+= Automatic Upgrades of Mule-App Projects to the Latest Mule Runtime
+
+After a new version of Mule runtime is released, opening an existing Mule-app project causes the application to use that new version. Immediately the application is upgraded, Design Center restarts it in the Design environment only. If Design Center is unable to upgrade an application, then the application is not restarted.
+
+This process ensures that Mule applications are always using the latest version of Mule Runtime.
+
+[IMPORTANT]
+====
+The upgrade might require you to update the connectors that are used in the applciation. Only after updating the connectors would be able to redeploy the application.
```

Comment:
Typo for "applciation" in the first sentence.

The word "you" is missing from the last sentence and the wording is awkward. I suggest:

The upgrade might require you to update the connectors that are used in the application. If so, you cannot redeploy the application until you update the connectors.

Reply:
Nice revision. Thanks!